modules/mod_ginger_base/templates/pager/pager.tpl

```diff
@@ -1,5 +1,9 @@
 <div id="{{ list_id }}-buttons" class="search__pager">
-    <div class="search__pager__result-counter">{% if items.total >= 30000 %}{_ More than 30000 results _} {% else %}{{ items.total }} {_ results _} {% endif %}</div>
+    <div class="search__pager__result-counter">
+        {% if items.is_total_estimated %}{{ _"About {n} results"|replace:"{n}":(items.total|round_significant|to_binary) }}
```

Comment:
We hebben ook [replace_args](https://zotonic.com/docs/1825/replace_args) toch? Dit is ook duidelijk, maar die is gemaakt voor deze use case. Verder kunnen de expressies na de `if` en de `else` misschien op aparte regels en geindent?

Reply:
Ik wil die `... {n} ...` gebruiken hier omdat de string dan compatible is met de 1.x (waar we tzt ook meervouden gaan gebruiken).